src/components/nodes/intermediateMessageThrowEvent/intermediateMessageThrowEvent.vue

```diff
@@ -24,11 +39,8 @@ export default {
         x: 2,
       },
     });
-    this.rootElements.push(this.message);
```

Comment:
This was the crux of the issue. Event if we don't end up using `this.message` it was being added to the BPMN process.